.npmignore

```diff
@@ -1,7 +1,8 @@
-**/*
-!README.md
-!LICENSE
-!PATENTS
-!package.json
-!typings/**/*
-!dist/**/*
\ No newline at end of file
+docs/
+src/
+test/
+theme/
+
+circle.yml
+ts*.*
```

Comment:
oh this was really confusing to read. can you please just list `tslint.json` and `tsconfig.json`